keras/distribution/distribution_lib.py

```diff
@@ -507,6 +529,28 @@ def _maybe_populate_device_mesh(self, layout):
 LayoutMap.get.__doc__ = LayoutMap.__getitem__.__doc__
 
 
+@keras_export("keras.distribution.distribute_tensor")
+def distribute_tensor(tensor, tensor_layout):
+    """Change the layout of a Tensor value in the jit function execution.
+
+    Note that this might not work outside of the jitted function for certain
+    backend. To change the layout of a value eagerly, please use
```

Comment:
It should work in both situations in JAX, right?

Reply:
It works for JAX, but might not for all the other backend, eg tf.dtensor.relayout() will only work with dtensor instance and also in a tf.function, and pytorch is also unknown for now. 

I think we will update the docstring once we have all the backend implemented, and reflect the final behavior.